tests/test_guided_transform.py

```diff
@@ -20,6 +45,55 @@ def transform_module():
     return TransformModule(TRANSFORM_MODULE_PATH)
 
 
+def test_standard_to_target_transform(caplog, dfs, transform_stage):
+    """
+    Test GuidedTransformer._standard_to_target transformation
+    """
+    # Pytest caplog fixture is set to WARNING by default. Set to INFO so
+    # we can capture log messages in GuidedTransformer._standard_to_target
+    caplog.set_level(logging.INFO)
+
+    # Transform
+    guided_transformer = transform_stage.transformer
+    target_instances = guided_transformer._standard_to_target(dfs)
+
+    # Check that output only contains concepts that had data and unique key
+    output_concepts = target_instances.keys()
+    expected_concepts = [c for c in output_concepts
+                         if c != 'sequencing_experiment']
+    assert not (set(output_concepts)
+                .symmetric_difference(set(expected_concepts)))
+
+    # Check instances counts and values
+    for target_concept, instances in target_instances.items():
+        # Only 2 unique participants
+        if target_concept == 'participant':
+            assert len(instances) == 2
+        else:
+            input_rows = dfs.get(target_concept).shape[0]
+            assert input_rows == len(instances)
+
+        for instance in instances:
+            assert instance.get('id')
```

Comment:
```suggestion
            assert 'id' in instance
```

Reply:
I want to test for existence of the key and a non-null value that's why i'm using get()